x/evm/types/params.go

```diff
@@ -20,6 +20,15 @@ import (
 	"github.com/evmos/evmos/v19/x/evm/core/vm"
 )
 
+const (
+	// Denom18Dec specifies that the evm denomination has 18 decimals
+	Denom18Dec = 18
+	// Denom6Dec specifies that the evm denomination has 6 decimals
+	Denom6Dec = 6
```

Comment:
To avoid possible consistency issues, we could wrap a new type like this:

```go
type DenomDecimals int
```

this way we can ensure no wrong or not supported decimals are used with DefaultDenomDecimals

Reply:
unfortunately Go does not really have enums though, so you could still pass e.g. 3 and it would be valid because it's a valid `int`

Reply:
Uhmm, right...

Reply:
it would be possible to do something like this though and then just call the `validate` method on the decimals value:

```go
type DenomDecimals int

func (d DenomDecimals) Validate() error {
	switch d {
	case Denom18Dec || Denom6Dec:
		return nil
	}

	return fmt.Errorf("invalid denomination decimal units: %d", d)	
}